benchmark/clblast/blas1/asum.cpp

```diff
@@ -42,7 +42,7 @@ void run(benchmark::State& state, ExecutorType* executorPtr, index_t size,
   state.counters["n_fl_ops"] = 2 * size_d;
   state.counters["bytes_processed"] = size_d * sizeof(scalar_t);
 
-  using data_t = utils::data_storage_t<scalar_t>;
+  using data_t = scalar_t;
```

Comment:
Why this change? This was originally done to support multiple data types, like half.

Reply:
I don't think clblast/clblas have support for that. If this is required more refactoring will be needed. I will double check if you can use half in clblast

Reply:
So, clBlast doesn't have TRSM with half support...

Reply:
BLAS implementations generally only support `float` and `double`, this is only to compare SYCL-BLAS to a reference implementation. Maybe then we don't need to enable it in this benchmark, but this particular change doesn't seem necessary, the original line should still work, right?

Reply:
In this case I had to change this because the `data_storage_t` is part of the internal utils of sycblas, located in `include/utils/quantization.hpp`, since I'm not linking the target `sycl_blas` into the clblast benchmarks, `utils::data_storage_t` will not be available here. Note that this will still be available in syclblas benchmarks.

Reply:
I see. I think we'll need to revisit this at a later point, though it's possible we don't have to do anything else.